django_mongodb_backend/fields/embedded_model_array.py

```diff
@@ -141,8 +141,47 @@ def as_mql(self, compiler, connection):
         return {"$anyElementTrue": lhs_mql}
 
 
+class ArrayAggregationSubqueryMixin:
+    def get_subquery_wrapping_pipeline(self, compiler, connection, field_name, expr):
+        return [
+            {
+                "$facet": {
+                    "group": [
```

Comment:
```suggestion
                    "group": [
```
Let's name it something else. Since group is an existing predicate, it could get confusing. How about `array_agg_query_mixin`. Just be as explicit as that

Reply:
🤔 IMO naming with some with meaning in the context of the pipeline is better. I am thinking in something like `gathered_values`, or `concatenated_values`.